packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +151,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+        }
+        .day,
+        .interact .day,
+        .week-day {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+            fill: currentcolor;
+            border-color: ${SystemColors.Canvas}
+        }
+        .interact .day:not(.disabled, .inactive):hover {
+            background: ${SystemColors.ButtonFace};
+            color: ${SystemColors.ButtonText};
+            outline: 1px solid ${SystemColors.Highlight};
+        }
+        .week-day:${focusVisible},
+        .interact .day:not(.disabled, .inactive):${focusVisible} {
+            forced-color-adjust: none;
+            background: ${SystemColors.ButtonFace};
+            border-color: transparent;
+            ${focusTreatmentBase}
+            outline-color: ${SystemColors.Highlight}
+        }
+        .day.disabled,
+        .day.inactive {
+            color: ${SystemColors.GrayText};
+            opacity:1;
+            outline: none;
+        }
+        .day.inactive:${focusVisible} {
+            outline: calc(${focusStrokeWidth} * 1px) solid ${SystemColors.GrayText};
+            box-shadow: none;
+        }
+        .day.selected,
+        .day.selected:not(.disabled, .inactive):hover {
+            color: ${SystemColors.Highlight};
+            outline: 1px solid ${SystemColors.Highlight}
```

Comment:
Same. Avoid `outline` for anything other than focus state.

Reply:
switched to border-color